### PR TITLE
Add integration test for Vector GPU

### PR DESCRIPTION
**Pull request by ThomasRaoux**

### CODE REVIEW

**Comment by ThomasRaoux** [comment]
Only the second commit is the test first commit is from https://github.com/google/iree-llvm-sandbox/pull/202

**Comment by ThomasRaoux** [comment]
Is there an easy way to disable the gpu integration test when building without gpu support?

File: configure.py [hunk 1]
```diff
@@ -67,6 +67,13 @@ def parse_arguments():
       default=False,
       action=argparse.BooleanOptionalAction,
   )
+  parser.add_argument(
+      "--cuda-runner",
```
Comment (matthias-springer) [inline configure.py:71]: What happens when I run `configure.py` without this argument and then run all tests? (run_tests.py)

**Review by matthias-springer — APPROVED** [review]

File: configure.py (hunk 1 above)
Comment (ThomasRaoux) [reply configure.py:71]: Good point, I talked to Nicolas about that and I do need to change the script to only enable cuda tests on demand

**Comment by ThomasRaoux** [comment]
@nicolasvasilache @matthias-springer I rebased and added a command line option to enable gpu integration tests since they require a gpu and cuda setup. Let me know if this looks okay to you.

**Pull request merged**